ODIN_II/SRC/simulate_blif.cpp

```diff
@@ -1175,6 +1181,13 @@ int get_clock_ratio(nnode_t* node) {
     return node->ratio;
 }
 
+/**
+ * Checks if the pin is one of the drivers of the provided net
+ */
+static bool pin_is_driver(npin_t* pin, nnet_t* net) {
```

Comment:
appologies, after careful review, the two places you use this, the loop actually serves to verify that the foward and back pointers are alligned

Reply:
Ok np